docker-images/artifacts/kafka-thirdparty-libs/2.8.x/pom.xml

```diff
@@ -16,7 +16,7 @@
     </licenses>
 
     <properties>
-        <strimzi-oauth.version>0.8.1</strimzi-oauth.version>
+        <strimzi-oauth.version>0.10.0-rc1-pre</strimzi-oauth.version>
```

Comment:
Will this work with Kafka 2.8.0? I remember we did changes to the authorizer part in 3.0.0.

Reply:
It will work, but it does slightly change the logging. So if someone is using `keycloak` authorization and is parsing the Kafka logs applying some regexes, then their regexes may no longer catch what they were catching before. Upgrading to 3.0.x would result in the same change. Thus in previous version there was a slight difference in logging when using `keycloak` authorization with 2.8.x vs. 3.0.x. By using the new oauth version on both that difference will now be gone. And it is necessary anyway in order to provide the new features (client timeouts, group extraction).

Reply:
> It will work, but it does slightly change the logging. So if someone is using keycloak authorization and is parsing the Kafka logs applying some regexes, then their regexes may no longer catch what they were catching before.

Thanks, If it is only logging, I don't think that would be a problem. At least not for me :-o.